snippets/src/offerCreate-v2.ts

```diff
@@ -0,0 +1,16 @@
+const client = new WsClient({server: `wss://xrpl.ws`})
+const wallet: Wallet = Wallet.walletFromSeed("sEd…")
+
+const tx: OfferCreateTransaction = {
+    account: "r…",
+    taker_gets: "XRP",
+    taker_pays: {...}
+}
+
+const preparedTransaction = client.autofill(tx)
+const signedTransaction = sign(wallet, preparedTransaction)
+
+const info: SubmitResponse = client.request({
```

Comment:
I prefer the signature of the existing request method over this one, but it's fine to experiment + try this out too, and compare.

Reply:
I want to try this out in Typescript, and then decide which one I like better. 

IMO this is better because it exactly mirrors the rippled websocket API.